### PR TITLE
add a bash script that invokes biotransformer

### PR DESCRIPTION
**Pull request by martenson**
after changing into the directory where the data is.
Without having the support data accessible in cwd
the biotransformer will fail.

trying to work around 
```
java.io.FileNotFoundException: supportfiles/CYP1A2/supportfile.csv (No such file or directory)
```

idea of @nsoranzo 

edit: ~~based on the failed test I have a suspicion that conda will not allow this~~ looks promising

supersedes https://github.com/bioconda/bioconda-recipes/pull/24302

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/biotransformer/build.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@ outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
 cp -R * $outdir/
-cp $RECIPE_DIR/biotransformer.py $outdir/biotransformer
+cp $RECIPE_DIR/biotransformer.py $outdir/biotransformer.py
```
Comment (nsoranzo) [inline recipes/biotransformer/build.sh:8]:
```suggestion
cp $RECIPE_DIR/biotransformer.py $outdir/
```

File: recipes/biotransformer/build.sh [hunk 2]
```diff
@@ -5,7 +5,10 @@ outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
 cp -R * $outdir/
-cp $RECIPE_DIR/biotransformer.py $outdir/biotransformer
+cp $RECIPE_DIR/biotransformer.py $outdir/
 ls -l $outdir
+printf '#!/bin/bash\n' > $outdir/biotransformer
+printf "cd ${outdir} \n" >> $outdir/biotransformer
+printf 'python biotransformer.py "$@"\n' >> $outdir/biotransformer
```
Comment (nsoranzo) [inline recipes/biotransformer/build.sh:12]: I was thinking that the `cd` could break passing relative paths to `biotransformer`, would that work instead?

```suggestion
printf 'python ${outdir}/biotransformer.py "$@"\n' >> $outdir/biotransformer
```

Comment (martenson) [reply recipes/biotransformer/build.sh:12]: @nsoranzo in my limited testing `${outdir}` will not expand when in single quotes

Comment (nsoranzo) [reply recipes/biotransformer/build.sh:12]: I haven't added single quotes, sorry I don't understand.

Comment (martenson) [reply recipes/biotransformer/build.sh:12]: @nsoranzo  but you added the `${outdir}` expansion and we are within the single quotes. Or am I missing something?

Comment (martenson) [reply recipes/biotransformer/build.sh:12]: Aside from the single quotes, in my testing the python script called from bash with full path will still operate on cwd (not the directory where the .py script is).

iow the following code with fail unable to find the `/Users/marten/miniconda3/share/biotransformer-1.1.5-0/supportfiles/` folder.

```
#!/usr/bin/env bash
python /Users/marten/miniconda3/share/biotransformer-1.1.5-0/biotransformer "$@"
```

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by martenson** [comment]
The package works alright in my testing. It has to be passed an absolute path though, like @nsoranzo warned.

**Comment by martenson** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by martenson** [comment]
Thanks @bgruening!

**Pull request merged**